### DISCUSSION TITLE
R 中的 apply 族函数 | /project/r-apply/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#615] [Announcements]
https://yuanfan.rbind.io/project/r-apply/

**Comment by hongtaoh** [comment]
这几个函数我现在都傻傻分不清。

**Reply by earfanfan** [reply]
我在写这篇笔记之前也分不清，而且上网搜看别人写也是说这几个函数很难，可是等我耐心地梳理完以后，觉得没有网上说的那么夸张。

**Comment by xiaole12138** [comment]
是我目前为止看到对于这几个函数的最棒的解释，希望可以在多出几个关于R中函数的解释，加油！

**Reply by earfanfan** [reply]
嗯，这位朋友，我收到了你的夸奖，谢谢你，今年还会继续写写 R 函数的，请放心地期待着，哈哈。

**Comment by Solarat** [comment]
写的非常好，看了之后，对这些函数有了更好的理解
